content/development/intro-haiku-ecosystem.md

```diff
@@ -0,0 +1,23 @@
++++
+type = "article"
+title = "Haiku Ecosystem"
+date = "2018-10-19T19:41:00.000Z"
+tags = ["Development","Ecosystem","ports","HaikuArchives","source","code"]
++++
+
+<p class="intro">Haiku is composed of the core Operating System itself, Haiku apps under the Haiku Archives project, and applications ported from other operating system held by the Haiku Ports project.</p>
+
+<a name="haiku"></a>
+<h3>The Haiku Operating System</h3>
+<p>*Haiku source code:* https://github.com/haiku/haiku</p>
+<p>Known simply as "Haiku" rather than "Haiku OS", the core Operating System consists of the Kernel and built in Haiku native applications that are always installed with the operating system. This includes everything from device drivers all the way up to system settings apps, the Tracker file browsing app, the Deskbar that allows launching and switching between apps, all the way up to the downloadable Haiku anyboot images for installing Haiku.</p>
```

Comment:
I was thinking of having that there because it is part of the build of the OS - i.e. creating an anyboot image is a build process. Because of this, it lives logically in the OS part of the docs. How to use the image is a User Guide topic, but building it belongs here I think.